tests/integration/targets/connection_wsl/plugin-specific-tests.yml

```diff
@@ -0,0 +1,32 @@
+---
+# Copyright (c) Ansible Project
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+- hosts: "{{ target_hosts }}"
+  gather_facts: false
+  serial: 1
```

Comment:
Is this necessary? If so, why?

Reply:
I'm afraid I do not known. I've removed it, and will see whether the tests still succeed.